docs/en/02_Developer_Guides/05_Extending/04_Shortcodes.md

```diff
@@ -56,7 +56,7 @@ First we need to define a callback for the shortcode.
 			'MyShortCodeMethod' => 'HTMLText'
 		);
 
-		public function MyShortCodeMethod($arguments, $content = null, $parser = null, $tagName) {
+		public static function MyShortCodeMethod($arguments, $content = null, $parser = null, $tagName) {
```

Comment:
Good spot, if it's a static method it should follow our naming conventions and be `my _shortcde_method`. Would you mind updating the docs to use that name?